apps/ui/src/helpers/intercom.ts

```diff
@@ -0,0 +1,47 @@
+const APP_ID = import.meta.env.VITE_INTERCOM_APP_ID;
+
+export function startIntercom() {
```

Comment:
Are we using correct script there?

From https://developers.intercom.com/installing-intercom/web/installation/

It seems we are using standard method:
> If you have a web app with multiple pages where each one triggers a new page refresh then you will most likely need the basic JavaScript method. This means you will not need to create separate actions which trigger when the Messenger will appear or update.

But there is SPA method available as well:
> If you have a single page app then you can get Intercom installed on your page in two simple steps. So let's waste no time and get you chatting to your customers.